packages/web-components/fast-components/CONTRIBUTING.md

```diff
@@ -4,17 +4,51 @@
 
 The component explorer is a [site](https://explore.fast.design) that showcases the components in the `@microsoft/fast-components` package. This acts as part of the documentation story, in order to expose a component it must have a [definition](#definition) and [configuration](#configuration).
 
-## Definition
+## Definitions
 
-Each component must have a **definition**. This file describes the components API.
-
-Steps to add a new definition:
-- Create a TypeScript file using spinal-case, this should be the name of your component and append it with `definition` which should be preceeded by a `.`. An example would be a `fast-button`, the expected name would be `fast-button.definition.ts`.
-- Add your export from the file to the component's `index.ts`.
+### Component Definition
 
-## Open UI Definition
+Each component must have a **definition**. This file describes the components API.
 
-Each component must have an [Open UI definition](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-components/src/__test__/component.schema.json). The `implementation` section of the definition will be populated by the above definition.
+First create a `.ts` (typescript) file using spinal-case. This should be the name of your component, append with `.definition`:
+```
+my-component/
```

Comment:
This isn't so much related to the change, but we don't really describe what the definition *is* anywhere... What is this definition export? Is there a description we can link to?

This might mostly be a question for @janechu

Reply:
There is a very brief description on line 11. It could be elaborated on.

Reply:
We could link to the VSCode customData github for reference as that is where the format for the definition comes from: https://github.com/microsoft/vscode-custom-data

You might include a line like "in order to be used in @microsoft/fast-tooling and for integration with the monaco editor the definition used is the same as the VSCode customData format".

Reply:
@nicholasrice @janechu Updated to include more clarity for Definitions. Please review when you get a chance. Thanks!